_episodes/08-Python.md

```diff
@@ -0,0 +1,252 @@
+This second instruction tells the computer to add each of those numbers in the list ('a') together and give that total a value of 'x'.
+
+## Print
+
+In the SAME cell type the following underneath:
+
+~~~
+print(x)
+~~~
+{: .language-python}
+
+Click on 'Run' or use the shortcut **Shift+Enter** to execute the cell.
+
+This last instruction tells the computer to print the total of the list of numbers on the screen.
+
+You just did some computing! Hooray!
+
+> ## Activity
+>
+> Take a few minutes to change the range of numbers, and/or change the values and see what happens. Have a bit of fun with it - see if you can beat the computer with your lightning speed mental arithmetic skills. Or just be amazed at how fast it can be.
+{: .challenge}
+
+## Creating a list
+
+Lists are ordered sequences of elements, and values can be repeated. To make a list you use square brackets.
+
+In your 'code' cell, write the following series of commands:
+
+~~~
+arr = ["Jack", "Queen", "King"]
+print(arr[0])
+print(arr[1])
+print(arr[2])
+~~~
+{: .language-python}
+
+Click on 'Run' or use the shortcut **Shift+Enter** to execute the cell.
+
+The command `arr=` gives the content of the square brackets the value of an 'array'. Each thing in the list has then an automatically appointed number, from its order in the list.
+
+> ## Tip
+>
+> Python uses [zero-based numbering](https://en.wikipedia.org/wiki/Zero-based_numbering),  which assigns the first item in a list the position of 0, rather than 1, as is typical in everyday non-programming situations.
+{: .callout}
+
+Given that the first number in a list in Python is 0, in this case the word "Jack" is position 0, the second, "Queen" is position 1, and so on.
+
+Those numbers are then what you use to perform your computation or visualisation, as you see when you use the command 'print' to show each one as a printed output on the screen. What you can see is that you have assigned each word a number (also known as a key, or array index).
+
+> ## Activity
+>
+> What would you do to print the items in the list in a different order?
+>
+> (HINT: There are more ways than one!)
+{: .challenge}
+
+## Creating a dictionary or set
+
+Now let's create a dictionary or set, using data of different types. In this table we'll use text, number and date data types.
+
+Dictionaries (or sets) are collections of unique elements and you cannot order them.
+
+Remember: Lists are ordered sequences of elements, and values can be repeated. Lists use square brackets.
+
+Curly braces are used in Python to define a dictionary. A dictionary is a data structure that maps one value to another - kind of like how an English dictionary maps a word to its definition.
+
+In your 'code' cell, write the following series of commands:
+
+~~~
+d = {'employee': 'Juanita Lopez','salary':81000, 'startdate': '2010-11-1'}
+e = {'employee': 'Peter Gynn','salary':83400, 'startdate': '2008-3-25'}
+f = {'employee': 'Jolie Talofa','salary':96800, 'startdate': '2007-3-14'}
```

Comment:
```suggestion
d = {'employee': 'Juanita Lopez', 'salary': 81000, 'startdate': '2010-11-1'}
e = {'employee': 'Peter Gynn', 'salary': 83400, 'startdate': '2008-3-25'}
f = {'employee': 'Jolie Talofa', 'salary': 96800, 'startdate': '2007-3-14'}
```

Reply:
I'm not sure whether using single-letter variable names is a good practice we want to encourage ;)

Reply:
I don't know how to code so I'm not sure what a good alternative is here. I am very happy to hear your suggestions!

Reply:
OK so I am going with replacing the letters with the first names of the employees. I'm also replacing 'k' with 'key' and v with 'value'.